### ISSUE TITLE
Gloot fails to return string property started from number

### ISSUE THREAD

**Issue by Nordsoft91** [peter-kish/gloot#212]
item protoset
```json
{
	\"id\": \"snack_food\",
	\"description\": \"150 kcal\"
},
```

code
```
var descr: String = item.get_property("description", "")
```

Expected result:
- descr variable has string type and value "150 kcal"

Actual result:
- get_property returns int = 150

<img width="416" alt="Снимок экрана 2024-05-24 в 14 41 03" src="https://github.com/peter-kish/gloot/assets/9308612/be3abf0c-03b4-4d6a-ac27-9c7ee748e925">

**Comment by peter-kish** [comment]
I did a bit of investigation and this seems like a Godot problem. `str_to_var` returns a variable of type int whenever a string begins with a digit. This behavior is also inconsistent with `String.is_valid_int()`:

```gdscript
var s: String = "1a"
print(s.is_valid_int()) # prints false
print(typeof(str_to_var(s)) == TYPE_INT) # prints true
```
I'll probably open an issue in the Godot repo.

**Comment by peter-kish** [comment]
Should be fixed in [v2.4.9](https://github.com/peter-kish/gloot/releases/tag/v2.4.9).
